galasa-parent/build.gradle

```diff
@@ -34,6 +25,21 @@ dependencies {
     compileOnly 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.41'
 }
 
+allprojects {
+   tasks.withType(Javadoc) {
+      options.addStringOption('Xdoclint:none', '-quiet')
+   }
+
+    repositories {
+        gradlePluginPortal()
+        mavenCentral()
+        maven {
+            url "https://development.galasa.dev/main/maven-repo/obr/"
+        }
+        mavenLocal()
```

Comment:
This ordering is important. I've previously fixed an issue in a different repo where local maven wasn't getting a look-in. It should be above the hosted maven I think ? may not make a difference normally tho.

Reply:
Agreed - moved `mavenLocal()` above hosted maven